requirements.txt

```diff
@@ -16,3 +16,4 @@ pyproj==3.7.0
 python-dateutil==2.9.0.post0
 rtree==1.3.0
 s1_orbits==0.1.1
+mypy
```

Comment:
Let's move `mypy` to `requirements-develop.txt`, which gets installed with the mamba environment.
```suggestion
```